spec.bs

```diff
@@ -312,9 +312,18 @@ To get the <dfn>max batch size</dfn> for an [=/origin=] |issuer|, run the follow
 
 1. TODO
 
-To <dfn>generate masked tokens</dfn> given a number |numTokens|, run the following steps:
+To <dfn>generate masked tokens</dfn> given [=struct=] |issuerKeys| (as described in [[#issuer-public-keys]]) and number |numTokens|, run the following steps:
```

Comment:
Is this a `[=key commitment=]` as described in https://github.com/WICG/trust-token-api/pull/207#discussion_r1119444546?